test/test.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
+
+// load plugins
+require('bedrock-server');
+require('bedrock-document-server');
+
+bedrock.events.on('bedrock.init', () => {
```

Comment:
This is not a pattern we've been using.  Is this addressing some difficulty?

Reply:
The difficulty is I threw this together quickly.  It's the same pattern used elsewhere, it just has the jsonld doc loader part pulled out.  Might be able to just remove this.